.github/workflows/fuzzers.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           - aflplusplus
           - aflplusplus_optimal
           - aflplusplus_shmem
+          - aflplusplus_optimal_shmem
```

Comment:
I'm confused by this? Isn't this already in fuzzers.yml?

Reply:
@vanhauser-thc